core/controllers/contributor_dashboard_admin.py

```diff
@@ -516,7 +516,7 @@ class ContributorDashboardAdminStatsHandlerNormalizedPayloadDict(TypedDict):
     language_code: Optional[str]
     sort_by: Optional[str]
     topic_ids: Optional[List[str]]
-    max_days_since_last_activity: Optional[int]
+    last_date: str
```

Comment:
Ok, I'm not sure now we need to change anything on the backend side. The parameters are almost the same, it's essentially a refactoring, not a new functionality. We can allow users to choose a date, but then we could calculate the number of days between now and that date and then pass it to the backend. 

Unless somebody disagrees with that, and unless you are fixing some bugs, I'd prefer keeping the backend as it is right now.

Reply:
Hello @nikitaevg, as per suggestion I will be leaving the backend as it is for now is the field name `max_days_since_last_activity` good to use ATM or should it be changed.

Reply:
done